### PR TITLE
Added Skip and Take feature 

### PR DESCRIPTION
**Pull request by DreamRunnerMoshi**
After order by, `take` and `skip` can be added now to generate t-sql like this.

`ORDER BY SOME_COLUMN
OFFSET 10 ROWS       -- To skip 10 rows`
`FETCH NEXT 10 ROWS ONLY;  -- take 10 rows`.

### CODE REVIEW

File: SqlQueryBuilder/Select/SelectQueryBuilder.cs [hunk 1]
```diff
@@ -18,6 +18,7 @@ public class SelectQueryBuilder : IQueryBuilderSelectFrom, IQueryBuilderJoinOrSe
         private List<string> JoinClauses = new List<string>();
         private List<string> OrderByClauses = new List<string>();
         private List<string> GroupByClauses = new List<string>();
+        private List<int> SkipTakeClauses = new List<int>();
```
Comment (Rem0o) [inline SqlQueryBuilder/Select/SelectQueryBuilder.cs]: Could use a better type to store the skip and take values, maybe a ValueTuple ? ( int, int )
```suggestion
        private List<int> SkipTakeClauses = new List<int>();
        // becomes
        private (int skip, int take) SkipTake =  (0, 0);
```

File: SqlQueryBuilder/Select/IQueryBuilderOrderBy.cs [hunk 2]
```diff
@@ -6,5 +6,6 @@ namespace SqlQueryBuilder.Select
     public interface IQueryBuilderOrderBy: IBuildQuery
     {
         IQueryBuilderOrderBy OrderBy<T>(Expression<Func<T, object>> lambda, bool desc = false, string tableNameAs = null);
+        IQueryBuilderOrderBy SkipTake(int skip, int take);
```
Comment (Rem0o) [inline SqlQueryBuilder/Select/IQueryBuilderOrderBy.cs]: Wouldn't it be better if Skip and Take were 2 different individual signatures? You might want to only use skip and then select all after those.

**Review by Rem0o — CHANGES_REQUESTED** [review]
Great addition, but there's 2 things I would like to be addressed ( see comments ).

File: SqlQueryBuilder/Select/SelectQueryBuilder.cs (hunk 1 above)
Comment (DreamRunnerMoshi) [reply SqlQueryBuilder/Select/SelectQueryBuilder.cs]: `IQueryBuilderOrderBy Skip(int skip);
 IQueryBuilderOrderBy Take(int take);`
 I seperated into `Skip` and `Take` two different methods.

Comment (Rem0o) [reply SqlQueryBuilder/Select/SelectQueryBuilder.cs]: Perfect!

**Review by Rem0o — APPROVED** [review]

**Pull request merged**